tests/numpy/columns/test_bool.py

```diff
@@ -0,0 +1,68 @@
+try:
+    import numpy as np
+except ImportError:
+    np = None
+
+from tests.numpy.testcase import NumpyBaseTestCase
+
+
+class BoolTestCase(NumpyBaseTestCase):
+    n = 10
```

Comment:
You should add

```python
required_server_version = (X, Y, Z)
```

for skipping this testcase on old servers.

Reply:
Should be resolved